qa/rpc-tests/wallet_z_sendmany.py

```diff
@@ -321,20 +322,25 @@ def run_test(self):
 
         # If we try to send 3 ZEC from n1ua0, it will fail with too-few funds.
         recipients = [{"address":n0ua0, "amount":3}]
-        linked_addrs_msg = 'Insufficient funds: have 2.00, need 3.00. (This transaction may require selecting transparent coins that were sent to multiple Unified Addresses, which is not enabled by default because it would create a public link between the transparent receivers of these addresses. THIS MAY AFFECT YOUR PRIVACY. Resubmit with the `privacyPolicy` parameter set to `AllowLinkingAccountAddresses` or weaker if you wish to allow this transaction to proceed anyway.)'
-        opid = self.nodes[1].z_sendmany(n1ua0, recipients, 1, 0)
+        linked_addrs_msg = 'Insufficient funds: have 2.00, need 3.00; note that coinbase outputs will not be selected if you specify ANY_TADDR, any transparent recipients are included, or if the `privacyPolicy` parameter is not set to `AllowRevealedSenders` or weaker. (This transaction may require selecting transparent coins that were sent to multiple Unified Addresses, which is not enabled by default because it would create a public link between the transparent receivers of these addresses. THIS MAY AFFECT YOUR PRIVACY. Resubmit with the `privacyPolicy` parameter set to `AllowLinkingAccountAddresses` or weaker if you wish to allow this transaction to proceed anyway.)'
+        opid = self.nodes[1].z_sendmany(n1ua0, recipients, 1, 0, 'AllowRevealedAmounts')
         wait_and_assert_operationid_status(self.nodes[1], opid, 'failed', linked_addrs_msg)
```

Comment:
I was just looking at this same check for a different reason …

I think this check is poorly defined. Basically, there are multiple things “wrong” with this tx, and because of changes in this PR the error that comes back given the default policy (`FullPrivacy` in this case) has changed from “Insufficient Funds” to “Could not select a unified address receiver […] without publicly revealing the transaction amount.” So I added the explicit policy to get the original error back, but perhaps this check _should_ be more about making sure the default policy fails, regardless of error message.

I.e., unless you say otherwise, I’ll change this to use the default policy, which would make it have the same error as line 331 below, but is distinct because it being `FullPrivacy` is implicit.